ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/experiencefragment/v1/experiencefragment/README.md

```diff
@@ -0,0 +1,49 @@
+<!--
+Copyright 2021 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Commerce Experience Fragment (v1)
+
+The Commerce Experience Fragement component is a server-side component written in HTL, allowing to dynymically display an experience fragment based on:
+* the product or category identifier defined in the page URL
```

Comment:
Currently it only works for product pages. So I assume you want to use the same component for implementing CIF-1683, right?

Reply:
You're right about CIF-1683, I removed the mention to categories and will add this in a future PR.